server/chat-commands/core.ts

```diff
@@ -539,7 +539,7 @@ export const commands: ChatCommands = {
 			return this.errorReply(this.tr`User ${targetUsername} is offline.`);
 		}
 
-		return this.parse(target);
+		return this.parse(target, this.isQuiet);
```

Comment:
Instead of passing `this.isQuiet` as an argument, should `this.parse` automatically inherit the `isQuiet` setting of the parent context?

Reply:
A potential issue with that would be hiding the help text if a command were used incorrectly.

Reply:
Fair enough. I think it should at least be an option argument rather than a positional argument.

Reply:
Updated.